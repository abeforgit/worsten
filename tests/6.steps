N
W
N
E
W
N
W