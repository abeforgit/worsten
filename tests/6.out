~ ~ ~ ~ ~ ~ 
~ + # # +∩~ 
~ + # # +∪~ 
~ + + + +E~x
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + # #∩+ ~ 
~ + # #∪+x~ 
~ + + + +N~ 
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + #∧# + ~ 
~ + #∨# + ~ 
~ + + +x+W~ 
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + #∧# + ~ 
~ + #∨# +x~ 
~ + + + +N~ 
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + #∧# + ~ 
~ + #∨# + ~ 
~ + + + +E~x
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + #∧# + ~ 
~ + #∨# + ~ 
~ + + +E+x~ 
~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ 
~ + #∧# + ~ 
~ + #∨#x+ ~ 
~ + + +N+ ~ 
~ ~ ~ ~ ~ ~ 

Opgelost!

